src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Starting the bot
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+        Task List[] = new Task[100];  //Creating a list of tasks
+        int listSize = 0;
+
+        while(true) {
+            System.out.println("--------------------------------------------------------------"); //Formatting
+            String line; //Input from user
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+
+            if(line.split(" ")[0].equals("bye")) {
+                break;
+            } else if(line.split(" ")[0].equals("list")){
+                for (int i = 0; i < listSize; i++) {
+                    System.out.println(i+1 + "." + List[i]);
+                }
+            } else if(line.split(" ")[0].equals("done")) {
+                //When the user keys in done
+                int taskNumber = Integer.valueOf(line.split(" ")[1]);
+                List[taskNumber-1].setDone("X");
+            } else if(line.split(" ")[0].equals("todo")) {
+                //When the user adds a todo
+                List[listSize] = new Todo(line.split("todo ")[1]);
+
+                System.out.println("Got it. I've added this task:");
+                System.out.println(List[listSize]);
+                System.out.println("Now you have " + (listSize + 1)  + " tasks in the list.");
+                listSize++;
+            } else if(line.split(" ")[0].equals("deadline")) {
```

Comment:
Notice how you have repeated `line.split(" ")[0].equals("something")` in every branch of this `if else` block. You can consider extracting `line.split(" ")[0]` as a variable at the start, and using a `switch case` to check for equality instead. This would make your code more readable as well